### PR TITLE
LPS-103595 Create a group for each AccountEntry to be used for assigning roles and permission checking

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-103595](https://issues.liferay.com/browse/LPS-103595).

Hey @pei-jung, can you look over my changes and make sure everything looks good or that I didn't miss anything.

The context for this is that in order to properly handle permissioning for account-related actions, we're going to need each AccountEntry to have an associated group. That way when we check permissions, we pass in the AccountEntry's `groupId` for scoping.

### CODE REVIEW

File: modules/apps/account/account-service/src/main/java/com/liferay/account/model/impl/AccountEntryImpl.java [hunk 1]
```diff
@@ -22,4 +34,45 @@
...
+	public Group getAccountEntryGroup() {
+		DynamicQuery dynamicQuery = _getGroupDynamicQuery();
+
+		List<Group> groups = GroupLocalServiceUtil.dynamicQuery(dynamicQuery);
+
+		if (ListUtil.isEmpty(groups)) {
+			return null;
+		}
+
+		return groups.get(0);
```
Comment (pei-jung) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/model/impl/AccountEntryImpl.java]: I think we can just call `GroupLocalServiceUtil.fetchGroup(long companyId, long classNameId, long classPK)` for this?

https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/service/GroupLocalServiceUtil.java#L488-L489

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79975. Thanks!

**Comment by pei-jung** [comment]
oops, sorry I didn't see that you already sent this to Brian. I've closed mine.

**Pull request closed without merging**